modules/serverless-creating-broker.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * serverless-using-brokers.adoc
+
+[id="serverless-creating-broker_{context}"]
+= Creating a broker
+
+To create a broker, you must create a `ServiceAccount` for each namespace and give that `ServiceAccount` the required RBAC permissions.
+
+.Prerequisites
+* Knative Eventing installed, which includes the `ClusterRole`.
+
+.Procedure
+. Create the `ServiceAccount` objects:
++
+----
+$ oc -n default create serviceaccount eventing-broker-ingress
+$ oc -n default create serviceaccount eventing-broker-filter
+----
+
+. Give those objects RBAC permissions:
++
+----
+$ oc -n default create rolebinding eventing-broker-ingress \
+  --clusterrole=eventing-broker-ingress \
+  --serviceaccount=default:eventing-broker-ingress
+$ oc -n default create rolebinding eventing-broker-filter \
+  --clusterrole=eventing-broker-filter \
+  --serviceaccount=default:eventing-broker-filter
+----
+
+. Create RBAC permissions granting access to shared configmaps for logging, tracing, and metrics configuration (optional):
++
+----
+$ oc -n knative-eventing create rolebinding eventing-config-reader-default-eventing-broker-ingress \
+  --clusterrole=eventing-config-reader \
+  --serviceaccount=default:eventing-broker-ingress
+$ oc -n knative-eventing create rolebinding eventing-config-reader-default-eventing-broker-filter \
+  --clusterrole=eventing-config-reader \
+  --serviceaccount=default:eventing-broker-filter
+----
++
+[NOTE]
+====
+These commands assume the shared Knative Eventing components are installed in the `knative-eventing` namespace. If you installed the shared Knative Eventing components in a different namespace, replace `knative-eventing` with the name of that namespace.
+====
+
+. Create the broker:
++
+----
+cat << EOF | kubectl apply -f -
```

Comment:
We should probably also use `oc` here as well (instead of `kubectl`)?

Reply:
Absolutely! Sorry I missed this one :)